.github/ISSUE_TEMPLATE/syntax_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Syntax Request
+about: Request adding a new syntax to bat.
+title: "[Syntax Request] "
+labels: new-syntax
+assignees: ''
+
+---
+
+\-------------------------------------------------------------------------------------------------
+**BEFORE YOU MAKE A REQUEST:**
+
+Are you looking to add a new syntax to use on one of your devices?
+Bat supports locally-installed language definitions. See the link below:
+
+https://github.com/sharkdp/bat#adding-new-syntaxes--language-definitions
+
+If you think adding this syntax would help others as well, please remove this header and continue.
+\-------------------------------------------------------------------------------------------------
+
+
+**Syntax:** syntax/language name
+
```

Comment:
This is great!

We could even add a paragraph that:
- kindly asks the user to check if that syntax is "popular" enough (we recently had a discussion about this somewhere). We could add a link to packagecontrol.io/ and add the 10,000 downloads mark as a guideline.
- has a reference to [this section](https://github.com/sharkdp/bat/tree/master/assets#syntax-highlighting-in-bat) in case somebody wants to open a PR as well.